Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM gcc:4.9.4
+# assumes this alread has git installed
+#RUN apt-get -y update && apt-get install -y git
+RUN mkdir /opt/externpro
+COPY externpro-16.01.1-gcc494-64-Linux.sh /opt/externpro/
+RUN cd /opt/externpro
+RUN chmod +x /opt/externpro/externpro-16.01.1-gcc494-64-Linux.sh
+RUN yes | /opt/externpro/externpro-16.01.1-gcc494-64-Linux.sh --prefix=/opt/externpro
+RUN dpkg -i /opt/externpro/externpro-16.01.1-gcc494-64-Linux/pkg/cmake-3.3.2-Linux-x86_64.deb
```

Comment:
I take it you're running this on ubuntu and not centos? The other ovsrpro stuff supports RPM but not DEB packages.